deepchem/feat/tests/test_roberta_tokenizer.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+from deepchem.feat.roberta_tokenizer import RobertaFeaturizer
+import pytest
+
+
+class TestRobertaFeaturizer(unittest.TestCase):
```

Comment:
I'm not sure how well pytest marks / unittest work well together. We should consider just using pytest style unless we can verify that the mark intermingling below works correctly

Reply:
I was having trouble with those. Not sure but IIRC I had to remove them for the pytests to run on my system.